pkgs/development/python-modules/aiorun/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, pytest
+, pytestcov
+, uvloop
+}:
+
+buildPythonPackage rec {
+  pname = "aiorun";
+  version = "2019.11.1";
+  format = "flit";
+
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "cjrh";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04p3sci6af6qqfkcqamsqhmqqrigcwvl4bmx8yv5ppvkyq39pxi7";
+  };
+
+  checkInputs = [
+    pytest
+    pytestcov
+    uvloop
+  ];
+
+  checkPhase = ''
+    pytest
```

Comment:
```suggestion
    HOME=$TMPDIR pytest
```
```
PermissionError: [Errno 13] Permission denied: '/homeless-shelter'
builder for '/nix/store/lmnxci3xvpp4qkyk0gihcyvvlv1pz4sr-python3.8-aiorun-2019.11.1.drv' failed with exit code 1
error: build of '/nix/store/frd56ynplxvr5c2m728kkl8z2qpasjjd-python3.7-aiorun-2019.11.1.drv', '/nix/store/lmnxci3xvpp4qkyk0gihcyvvlv1pz4sr-python3.8-aiorun-2019.11.1.drv' failed
```

Reply:
So we no longer need to `HOME=$(mktemp -d)`? This is nice!